core-api/src/main/java/com/optimizely/ab/notification/UpdateConfigNotification.java

```diff
@@ -0,0 +1,23 @@
+/**
+ *
+ *    Copyright 2019, Optimizely and contributors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package com.optimizely.ab.notification;
+
+/**
+ * UpdateConfigNotification signals a change in the current configuration.
```

Comment:
I was wondering if there is a way we can document or force implementers to remember to trigger this notification whenever they set config (like you do in `PollingProjectConfigManager.setConfig`)

Reply:
I can add documentation to the ProjectConfigManger interface. Beyond that we just need to #trust